server/classes/sound.js

```diff
@@ -10,5 +10,6 @@ export default class Sound {
     this.playbackRate = params.playbackRate || 1;
     this.channel = params.channel || [0, 1];
     this.looping = params.looping || false;
+    this.preserveChannels = params.preserveChannels || false;
```

Comment:
This can default to true, since it will only apply to new sounds. I think this behavior is more expected than the channel down mixing / adjusting behavior.

```suggestion
    this.preserveChannels = params.preserveChannels === false ? false : true;
```